googledrive-repository/pom.xml

```diff
@@ -327,8 +324,7 @@
                                 </goals>
                                 <configuration>
                                     <tasks>
-                                        <copy file="src/main/customResponseContextFiles/registerSlingshotCustomResponse/google-customResponse-context.xml"
-                                              tofile="src/main/amp/config/alfresco/subsystems/googledocs/drive/google-customResponse-context.xml"/>
+                                        <copy file="src/main/customResponseContextFiles/registerSlingshotCustomResponse/google-customResponse-context.xml" tofile="src/main/amp/config/alfresco/subsystems/googledocs/drive/google-customResponse-context.xml" />
```

Comment:
Why put onto one line (is this this just an auto-formatting change) ?

Same for other occurrences ...

Reply:
Yes, is auto-formatting change :(

Reply:
Thanks for the quick review  💃